### DISCUSSION TITLE
incluindo e lendo um texto em um site

### DISCUSSION THREAD

**Discussion by victorcaldeira** [frontendbr/forum#1777] [Perguntas e Respostas]
pessoal, estou desenvolvendo um site, e preciso que o usuário digite um texto em forma de denuncia e gostaria que esse texto fosse salvo e mostra-lo na pagina como um fórum.

**Comment by felquis** [comment]
Acho que você tem que começar com Form e Textarea, e submeter o form para o backend em PHP, C# ou NodeJS, ai no backend quando receber a submissão do formulário você pode armazenar o texto em um banco de dados na memória.

Ai quando recarrega o front-end, traz também essas submissões e mostra na página.

**Comment by victorcaldeira** [comment]
@felipefialho você sabe se teria uma forma de fazer isso sem backend?

**Comment by felquis** [comment]
Explica com mais detalhes como você quer que funcione.
